src/components/CustomAutocomplete.tsx

```diff
@@ -28,7 +28,8 @@ export default function AutocompleteFilter(
       open={isOpen && !limitReached}
       disabled={limitReached}
       onInputChange={(e, val) => setIsOpen(val.length > 0)}
-      renderTags={() => null}
+      forcePopupIcon={false}
```

Comment:
helps with the previous bug I had.
Possible [explanation.](https://stackoverflow.com/questions/61256685/material-ui-autocomplete-styling)

Reply:
What does `forcePopupIcon` do? I understand it does something with the up/down icon on the right but I couldn't make heads or tail of the API description when it says "Force the visibility display"

Reply:
I think besides visibility, the styles are also not forced in this case.